anchore_engine/services/policy_engine/__init__.py

```diff
@@ -350,6 +352,33 @@ def handle_feed_sync_trigger(*args, **kwargs):
     return True
 
 
+def handle_grypedb_sync(*args, **kwargs):
+    """
+    Calls function to run GrypeDBSyncTask
+
+    :param args:
+    :param kwargs:
+    :return:
+    """
+    system_user = _system_creds()
+    if "GrypeDBSyncTask" not in locals():
+        from anchore_engine.services.policy_engine.engine.tasks import GrypeDBSyncTask
+
+    logger.info("init args: {}".format(kwargs))
+    cycle_time = kwargs["mythread"]["cycle_timer"]
```

Comment:
is it possible for this to cause a `KeyError`?

Reply:
I copied the approach used by the other tasks in the file. Not familiar with how all the monitors are set up but I believe it pulls the value from the definition in the __monitors__  list: https://github.com/anchore/anchore-engine/pull/989/files#diff-52aad75ea5c30e794c66b3da25a701839c2a1cfb61877c9b9e218d8cfc096f5dR453